### PR TITLE
k8smeetup-shirdrn-pr-2017-8-15

### PR DESCRIPTION
**Pull request by shirdrn**
## Task table

https://docs.google.com/spreadsheets/d/1FDFCv9RK5nSMgLXhPrJ5k7r5QvHnNEFnXbvoFiM8v20/edit#gid=1015862274

## Preview links

- https://k8smeetup.github.io/docs/tasks/administer-cluster/dns-custom-nameservers
- https://k8smeetup.github.io/docs/tasks/administer-cluster/cpu-memory-limit
- https://k8smeetup.github.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods
- https://k8smeetup.github.io/docs/concepts/services-networking/connect-applications-service
- https://k8smeetup.github.io/docs/concepts/workloads/pods/init-containers
- https://k8smeetup.github.io/docs/concepts/workloads/controllers/garbage-collection
- https://k8smeetup.github.io/docs/concepts/services-networking/service
- https://k8smeetup.github.io/docs/concepts/services-networking/dns-pod-service
- https://k8smeetup.github.io/docs/concepts/policy/pod-security-policy
- https://k8smeetup.github.io/docs/concepts/overview/working-with-objects/kubernetes-objects
- https://k8smeetup.github.io/docs/concepts/workloads/controllers/daemonset
- https://k8smeetup.github.io/docs/concepts/workloads/controllers/cron-jobs
- https://k8smeetup.github.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases
- https://k8smeetup.github.io/docs/concepts/services-networking/network-policies

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5376)

### CODE REVIEW

**Comment by shirdrn** [comment]
@markthink Plz review the PR, thx.

**Comment by markthink** [comment]
@chenopis I approve of this PR.

File: cn/docs/concepts/overview/working-with-objects/kubernetes-objects.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+### 对象 Spec 与状态
+
+每个 Kubernetes 对象包含两个嵌套的对象字段，它们负责管理对象的配置：对象 *spec* 和 对象 *status*。*spec* 必须提供，它描述了对象的 *期望状态* —— 希望对象所具有的特征。*status* 描述了对象的 *实际状态*，它是由 Kubernetes 系统提供和更新。在任何时刻，Kubernetes 控制平面一直处于活跃状态，管理着对象的实际状态以与我们所期望的状态相匹配。
+
+
+
+例如，Kubernetes Deployment 对象能够表示运行在集群中的应用。当创建 Deployment 时，可能需要设置 Deployment 的 spec，以指定该应用需要有 3 个副本在运行。Kubernetes 系统读取 Deployment spec，启动我们所期望的该应用的 3 个实例 —— 更新状态以与 spec 相匹配。如果那些实例中有失败的（一种状态变更），Kubernetes 系统通过修正来响应 spec 和状态之间的不一致 —— 这种情况，启动一个新的实例来替换。
+
+关于对象 spec、status 和 metadata 更多信息，查看 [Kubernetes API Conventions](https://git.k8s.io/community/contributors/devel/api-conventions.md)。
```
Comment (tengqm) [inline cn/docs/concepts/overview/working-with-objects/kubernetes-objects.md]: metadata后面少一个"的"

Conventions应该翻译，例如”约定"

File: cn/docs/concepts/overview/working-with-objects/kubernetes-objects.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+
+
+### 描述 Kubernetes 对象
+
+当创建 KUbernetes 对象时，必须提供对象的 spec，用来描述该对象的期望状态，以及关于对象的一些基本信息（例如，名称）。当使用 KUbernetes API 创建对象时（或者直接创建，或者基于`kubectl`），API 请求必须在请求体中包含 JSON 格式的信息。**更常用的是，需要在 .yaml 文件中为 `kubectl` 提供这些信息**。 `kubectl` 在执行 API 请求时，将这些信息转换成 JSON 格式。
```
Comment (tengqm) [inline cn/docs/concepts/overview/working-with-objects/kubernetes-objects.md]: most often -> 大多数情况下

Comment (tengqm) [reply cn/docs/concepts/overview/working-with-objects/kubernetes-objects.md]: make the API request -> 发起API请求，这里不应翻译为“执行”，容易引起误解。

File: cn/docs/concepts/overview/working-with-objects/kubernetes-objects.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+这里有一个 `.yaml` 示例文件，展示了 KUbernetes Deployment 的必需字段和对象 spec：
```
Comment (tengqm) [inline cn/docs/concepts/overview/working-with-objects/kubernetes-objects.md]: 下面的`.yaml`文件示例展示了Kubernetes Deployment对象所需要的字段和对应的对象Spec。

File: cn/docs/concepts/overview/working-with-objects/kubernetes-objects.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+{% include code.html language="yaml" file="nginx-deployment.yaml" ghlink="/docs/concepts/overview/working-with-objects/nginx-deployment.yaml" %}
+
+一种创建 Deployment 的方式，类似上面使用 `.yaml` 文件，是使用 `kubectl` 命令行接口（CLI）中的 [`kubectl create`](/docs/user-guide/kubectl/v1.7/#create) 命令，传递 `.yaml` 作为参数。下面是一个示例：
```
Comment (tengqm) [inline cn/docs/concepts/overview/working-with-objects/kubernetes-objects.md]: 这里对原文的理解有问题。

使用类似于上面的`.yaml`文件来创建Deployment时，一种方式是使用...，将该`.yaml`文件作为命令参数。例如：

**Comment by tengqm** [comment]
@markthink have you reviewed the PR actually?

File: cn/docs/concepts/overview/working-with-objects/kubernetes-objects.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+title: 理解 Kubernetes 对象
+
+redirect_from:
+- "/docs/concepts/abstractions/overview/"
+- "/docs/concepts/abstractions/overview.html"
+---
+
+{% capture overview %}
+
+该页说明了 Kubernetes 对象在 Kubernetes API 中是如何表示的，以及如何在 `.yaml` 格式的文件中表示。
```
[the remark this answers is not in the document]
Comment (markthink) [reply cn/docs/concepts/overview/working-with-objects/kubernetes-objects.md]: this page 翻译成该页与本页，意义差别并不大。

**Comment by markthink** [comment]
@tengqm yes.

File: cn/docs/concepts/overview/working-with-objects/kubernetes-objects.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (tengqm) [reply cn/docs/concepts/overview/working-with-objects/kubernetes-objects.md]: this page -> 本页
that page -> 该页

**Comment by tengqm** [comment]
@markthink Okay then, I believe there are still a lot issues to be solved in this PR, if we are serious about producing high-quality documentation for users. No one would enjoy the feeling of being blamed for avoidable mistakes in the doc.

**Comment by shirdrn** [comment]
@tengqm @markthink 
Again, I have tried to translate and review all these docs in this PR, and I've fixed the translation issues that I could find. Please check and review, thanks.

**Comment by chenopis** [comment]
@shirdrn There seems to be a merge conflict w/ `cn/docs/concepts/services-networking/network-policies.md` now.

**Comment by shirdrn** [comment]
@chenopis I've resolved the merge conflict, plz help to merge this PR, thx.

**Pull request merged**